docs/releases/3.1.txt

```diff
@@ -96,6 +96,10 @@ Minor features
 * The password reset mechanism now uses the SHA-256 hashing algorithm. Support
   for tokens that use the old hashing algorithm remains until Django 4.0.
 
+* The :func:`~django.contrib.auth.hashers.make_password` now requires its
+  argument to be a string or bytes. Other types should be explicitly cast to
+  one of these, if required.
```

Comment:
I'm not sure what "if required" means? Isn't it now required?

Reply:
It was my (draft) suggestion on the ticket. The thought was "if you're using other types". But, yes, it's redundant 👍